### DISCUSSION TITLE
Widen overlay image on Splash of Home layout

### DISCUSSION THREAD

**Discussion by DJreyaB** [mmistakes/minimal-mistakes#2997] [Q&A] [answered]
Can someone point me in the direction of the css place to widen the overlay image for splash or home? I cannot find an image thin enough to fit the sliver. I'd prefer doubling the current allotted space, however I'm not sure which id goes with which item. If there is a document that directly states the correlation for id's I'd appreciate that as well. 

![Screen Shot 2021-06-15 at 3 25 33 AM](https://user-images.githubusercontent.com/37115281/122010705-5c702880-cd89-11eb-83d3-ff632ef7f366.png)

**Comment by mmistakes** [comment, marked as answer]
Duplicate of #542
